.github/workflows/docker.yml

```diff
@@ -101,10 +134,12 @@ jobs:
         with:
           context: ${{ matrix.context }}
           file: ${{ matrix.file }}
-          platforms: ${{ matrix.platforms }}
+          platforms: ${{ matrix.platform }}
```

Comment:
I believe the platforms need to be specified all at once here, otherwise the different jobs only build images for one single platform and overwrite eachother on push.

Reply:
Oh damn, I didn't realize that